core-bundle/contao/templates/twig/backend/jobs/_progress.html.twig

```diff
@@ -0,0 +1 @@
+<div class="jobs-progress" style="--value:{{ progress|round }}" aria-valuemin="0" aria-valuemax="100" aria-valuenow="{{ progress|round }}" data-label="{{ progress|format_number(2) }}%"></div>
```

Comment:
```suggestion
<div class="jobs-progress" style="--value:{{ progress|round }}" aria-role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="{{ progress|round }}" data-label="{{ progress|format_number(2) }}%"></div>
```

See: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/progress

Whilst being recommended to use the `<progress>` element, a styled div is fine with their aria-attributes. We should however mark this div's role as well `role="progressbar"`.

Reply:
Ah, I didn't even know that. Should we switch to `<progress>` or is that harder to style because of browser defaults?

Reply:
Not harder to style, I would even prefer using the progress bar, would save some styling :)

Reply:
After some tinkering, we would have to style the following pseudo-elements which differ from browser engines:
```
/** Safari / Background is chromium */
progress::-webkit-progress-bar {
    background-color: var(--body-bg);
    border-radius: 50px;
}

/** Chromium */
progress::-webkit-progress-value {
    background-color: var(--progress-running);
    color: var(--body-bg);
    border-radius: 50px;
}

/** Gecko engine */
progress::-moz-progress-bar {
    background-color: var(--progress-running);
    color: var(--body-bg);
    border-radius: 50px;
}
```

On top of that, we would want the `progress-finished` value as well.
I'd be for simply adding `role="progressbar"` instead and keep the styles as they are in this PR